openlibrary/templates/books/edit.html

```diff
@@ -7,7 +7,9 @@
 
 $:macros.HiddenSaveButton("addWork")
 
-<div id="contentHead" class="editFormHead">
+<div id="contentHead">
+    <!-- removed class="editFormHead" -->
+    <!-- $:macros.databarEdit(work) -->
```

Comment:
Please:
1. Remove these two lines
2. Delete the `editFormHead` from `helps-misc.less` (since it isn't user anywhere else)

Reply:
Made this change